oap-server/server-receiver-plugin/configuration-discovery-receiver-plugin/src/main/java/org/apache/skywalking/oap/server/recevier/configuration/discovery/AgentConfigurationsReader.java

```diff
@@ -54,6 +54,9 @@ public AgentConfigurationsTable readAgentConfigurationsTable() {
                 if (configurationsData != null) {
                     configurationsData.forEach((k, v) -> {
                         Map map = (Map) v;
+                        if (configurationsData.containsKey("default-config")) {
+                            ((Map) configurationsData.get("default-config")).forEach(map::putIfAbsent);
+                        }
```

Comment:
I am confused about this part. As `default-config` is a reserved key, and it shares the same structure as other services. Why do we have to change the init logic?
Also, we don't build `agentConfigurationsTable` properly(no UUID part).

Reply:
We need to add the missing configuration items of each service in default-config to the each service configuration, I think they should be similar to union relationship. And then we will traverse the map to generate UUID for each service